docs/components/card-catalog.md

```diff
@@ -2,39 +2,19 @@
 
 **KCardCatalog** - A grid view of KCards
 
-<script>
-function getItems(count) {
-  let myItems = []
-  for (let i = 0; i < count; i++) {
-    myItems.push({
-      title: "Item " + i,
-      description: "The item's description for number " + i
-    })
-  }
-  return myItems
-}
-
-const longItem = {
-  title: "A very long item",
-  description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas in tempus lorem, et molestie quam. Praesent sapien massa, posuere et volutpat nec, imperdiet a dui. Fusce non leo posuere, molestie neque et, posuere ex. Nullam euismod tortor in est sagittis iaculis. In sodales bibendum nulla nec feugiat."
-}
-
-export default {
-  data () {
-    return {
-      getItems,
-      longItem
-    }
-  }
-}
-</script>
+<KCardCatalog :items="getItems(5)" />
 
 
-<KCardCatalog :items="getItems(5)" />
 ```vue
 <KCardCatalog :items="items" />
 ```
 
+Pass a fetcher function to build a slot-able card catalog.
+
+```vue
+<KCardCatalog 
+   :fetcher="fetcher" />
```

Comment:
Can you move this up on the same line as the component name?

Reply:
Done!